src/webui/api/torrentscontroller.cpp

```diff
@@ -886,9 +886,10 @@ void TorrentsController::deleteAction()
 
     const QStringList hashes {params()["hashes"].split('|')};
     const bool deleteFiles {parseBool(params()["deleteFiles"], false)};
-    applyToTorrents(hashes, [deleteFiles](BitTorrent::TorrentHandle *const torrent)
+    const DeleteOption deleteOption = deleteFiles ? TorrentAndFiles : Torrent;
```

Comment:
maybe combine them since `deleteFiles` variable isn't much of use:
```c++
const DeleteOption deleteOption = parseBool(params()["deleteFiles"], false)
    ? TorrentAndFiles : Torrent;
```

Reply:
All done.